File: core/src/main/java/forpleuvoir/dhwu1a/core/event/message/StrangerMessageEvent.java
package forpleuvoir.dhwu1a.core.event.message;

import forpleuvoir.dhwu1a.core.message.StrangerMessage;
import forpleuvoir.dhwu1a.core.user.base.User;

import javax.annotation.Nullable;

/**
 * 陌生人消息事件
 *
 * @author forpleuvoir
 * <p>#project_name dhwu1a
 * <p>#package forpleuvoir.dhwu1a.core.event.message
 * <p>#class_name StrangerMessageEvent
 * <p>#create_time 2021/7/4 12:06
 */
public class StrangerMessageEvent extends MessageEvent<StrangerMessage> {

    public StrangerMessageEvent(StrangerMessage message) {
        super(message);
    }

    @Nullable
    @Override
    public User getUser() {
        return null;
    }
}

File: core/src/main/kotlin/forpleuvoir/dhwu1a/core/event/message/StrangerMessageEvent.kt
package forpleuvoir.dhwu1a.core.event.message

import forpleuvoir.dhwu1a.core.message.StrangerMessage
import forpleuvoir.dhwu1a.core.user.base.User

/**
 * 陌生人消息事件
 *
 * @author forpleuvoir
 *
 * #project_name dhwu1a
 *
 * #package forpleuvoir.dhwu1a.core.event.message
 *
 * #class_name StrangerMessageEvent
 *
 * #create_time 2021/7/4 12:06
 */
class StrangerMessageEvent(message: StrangerMessage) : MessageEvent<StrangerMessage>(message) {
    override val user: User?
        get() = null

}